packages/openid4vc/src/openid4vc-issuer/router/accessTokenEndpoint.ts

```diff
@@ -0,0 +1,161 @@
+import type { OpenId4VcIssuanceRequest } from './requestContext'
+import type { AgentContext } from '@aries-framework/core'
+import type { JWTSignerCallback } from '@sphereon/oid4vci-common'
+import type { NextFunction, Response, Router } from 'express'
```

Comment:
For now it is okay to leave it like this, but I feel that including express in here is not a good solution in the long run. There are many other options out there and especially when using other runtimes like Deno.

Reply:
Yes I agree, but as we're already using express for DIDComm I didn't see a good reason to do it different for oid4vc.